src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -238,7 +238,7 @@ function processInlineTag(
     currentStyle = currentStyle.withMutations(style => {
       const fontWeight = htmlElement.style.fontWeight;
       const fontStyle = htmlElement.style.fontStyle;
-      const textDecoration = htmlElement.style.textDecoration;
+      const textDecoration = htmlElement.style.textDecoration.split(' ');
```

Comment:
I might leave a comment like "text-decoration is css shorthand and can contain multiple decoration values"